public/docs/ts/latest/guide/dependency-injection.jade

```diff
@@ -336,47 +338,47 @@ include ../../../../_includes/_util-fns
-  adding a constructor that takes a `logger` parameter.
+  We'd apply the same *constructor injection* pattern,
+  adding a constructor that takes a `Logger` parameter.
 
   Here is the revision compared to the original.
-  
+
 +makeTabs(
-  `dependency-injection/ts/app/heroes/hero.service.2.ts, 
+  `dependency-injection/ts/app/heroes/hero.service.2.ts,
   dependency-injection/ts/app/heroes/hero.service.1.ts`,
-  null, 
+  null,
   `app/heroes/hero.service (v.2),
   app/heroes/hero.service (v.1)`)
 
 :marked
   The constructor now asks for an injected instance of a `Logger` and stores it in a private property called `_logger`.
-  We call that property within our `getHeroes()` method when anyone asks for heroes.
+  We call that property within our `getHeroes` method when anyone asks for heroes.
 
   <a id="injectable"></a>
-  ### Why *@Injectable*?
+  ### Why @Injectable?
   Notice the `@Injectable()` decoration above the service class.
   We haven't seen `@Injectable()` before.
-  As it happens, we could have added it to our first version`HeroService`.
+  As it happens, we could have added it to our first version of `HeroService`.
   We didn't bother because we didn't need it then.
 
   We need it now... now that our service has an injected dependency.
   We need it because Angular requires constructor parameter metadata in order to inject a `Logger`.
-  As [we mentioned earlier](#di-metadata), TypeScript *only generates metadata for classes that have a decorator*. .
-  
+  As [we mentioned earlier](#di-metadata), **TypeScript only generates metadata for classes that have a decorator.**
+
 .callout.is-helpful
-  header Always add @Injectable() 
+  header Always add @Injectable()
   :marked
     We recommend adding `@Injectable()` to every service class, even those that don't have dependencies
-    and, therefore, do not technically require it. Two reasons:
-  ol(style="font-size:inherit")
-    li <i>Future proofing</i>: no need to remember <code>@Injectable</code> when we add a dependency later.
-    li <i>Consistency</i>: all services follow the same rules and we don't have to wonder why a decorator is missing.
+    and, therefore, do not technically require it. Here's why:
+  ul(style="font-size:inherit")
+    li <b>Future proofing:</b> No need to remember <code>@Injectable</code> when we add a dependency later.
+    li <b>Consistency:</b> All services follow the same rules, and we don't have to wonder why a decorator is missing.
 .l-sub-section
   :marked
     The `HeroesComponent` has an injected dependency too. Why don't we add `@Injectable()` to the `HeroesComponent`?
-    
+
     We *can* add it if we really want to. It isn't necessary because the `HeroesComponent` is already decorated with `@Component`.
-    TypeScript generates metadata for *any* class with a decorator and *any* decorator will do.
+    TypeScript generates metadata for *any* class with a decorator, and *any* decorator will do.
```

Comment:
You are the comma queen.  I'm not sure why you want the reader to trip over that comma on the way to "and". ;-)